otter/convergence/service.py

```diff
@@ -237,8 +229,8 @@ def convergence_succeeded(scaling_group, group_state, servers, now):
         yield Effect(
```

Comment:
This is great catch @cyli. The test passes because it does not check about servers being updated. I'll add it.